packages/ui/src/machines/authenticator/defaultServices.ts

```diff
@@ -14,7 +17,28 @@ export const defaultServices = {
   async getAmplifyConfig() {
     const result = Amplify.getConfig();
     groupLog('+++getAmplifyConfig', 'result', result);
-    return result;
+
+    const cliConfig = result.Auth.Cognito;
+    const parsedLoginMechanisms = Object.entries(cliConfig.loginWith)
```

Comment:
Can we handle this (and the below) in `getAuthenticatorConfig`?

Reply:
Discussed offline, will punt on this for now